packages/_private/vue-demo/package.json

```diff
@@ -10,6 +10,7 @@
     "_build-only": "vite build",
     "type-check": "vue-tsc --noEmit -p tsconfig.app.json --composite false",
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "format.eslint": "pnpm lint --fix",
     "format": "prettier --write src/"
```

Comment:
Do we need prettier explicitly here?

Reply:
The reason for this is that the application is bootstrapped via `npm create vue@latest` and not altered anymore after that. I wanted it to be using the standard of the framework. Angular and React demos also have tooling that is coming from their respective starters, so I did not mind.